examples/onnx/yolov5-youtube/predictor.py

```diff
@@ -160,19 +166,21 @@ def postprocess(self, output):
     def predict(self, payload):
         in_path = download_from_youtube(payload["url"], self.input_size[1])
         out_path = f"{uuid.uuid1()}.mp4"
-        writer = FrameWriter(out_path, size=self.input_size)
-
-        for frame in frame_reader(in_path, size=self.input_size):
-            x = (frame.astype(np.float32) / 255).transpose(2, 0, 1)
-            # 4 output tensors, the last three are intermediate values and
-            # not necessary for detection
-            output, *_ = self.client.predict(x[None])
-            boxes, class_ids, confidence = self.postprocess(output)
-            overlay_boxes(frame, boxes, class_ids, self.labels, self.color_map)
-            writer.write(frame)
-
-        del writer
+
+        with FrameWriter(out_path, size=self.input_size) as writer:
+            for frame in frame_reader(in_path, size=self.input_size):
+                x = (frame.astype(np.float32) / 255).transpose(2, 0, 1)
+                # 4 output tensors, the last three are intermediate values and
+                # not necessary for detection
+                output, *_ = self.client.predict(x[None])
+                boxes, class_ids, confidence = self.postprocess(output)
+                overlay_boxes(frame, boxes, class_ids, self.labels, self.color_map)
+                writer.write(frame)
+
+        with open(out_path, "rb") as f:
+            output_bytes = f.read()
+
         os.remove(in_path)
-        # We cant remove out_path, so the deployment will run out of memory
-        # sooner or later
-        return FileResponse(out_path)
+        os.remove(out_path)
+
+        return output_bytes
```

Comment:
What do you think about changing this to

```python
buf = BytesIO(output_bytes)
return StreamingResponse(buf, media_type="video/mp4")
```

With the current version, the returned mime-type is `application/octet-stream`